tests/tests_selinux_modules_checksum.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Test management of SELinux modules
+  hosts: all
+  become: true
+  tasks:
+    - name: Execute the role and catch errors
+      when: ansible_distribution == "Fedora" or
+        (ansible_distribution in ["CentOS", "RedHat", "Rocky"] and
+        ansible_distribution_version is version("8.6", ">="))
```

Comment:
This new `local_semodule` Ansible module will only support RHEL 8.6 and later?  Is that because of the `selinux` and `semanage` python library dependencies?  Will users of RHEL 6, RHEL 7, and RHEL 8 still be able to manage their selinux modules with this role?

Reply:
@bachradsusi ^^^ ?

Reply:
ok - then we need to add something to the README.md https://github.com/linux-system-roles/selinux#manage-selinux-modules - something like
"NOTE: Managing modules is idempotent only on Fedora, and EL 8.6 and later.  You can manage modules on old releases, but it will not be idempotent."